.github/ISSUE_TEMPLATE/new_metapackage.yml

```diff
@@ -58,30 +58,32 @@ body:
         - Delphi
         - Disassemblers
         - dotNet
+        - Documents
         - Evasion
         - Exploitation
+        - File Information
         - Forensic
         - Hex Editors
         - Information Gathering
         - InnoSetup
         - Java & Android
         - Javascript
         - Lateral Movement
+        - Memory
         - Networking
-        - Office
         - Packers
         - Password Attacks
         - Payload Development
-        - PDF
         - PE
         - Persistence
         - PowerShell
         - Privilege Escalation
-        - Python
+        - Productivity Tools
```

Comment:
are we planing to add some of the tools without category here @emtuls ? I think we should do it as a separate PR, this one is big enough :sweat_smile: